config/config.go

```diff
@@ -31,16 +36,26 @@ type IngressConfig struct {
 
 // Get returns an initialized IngressConfig
 func Get() *IngressConfig {
+	cfg := clowder.LoadedConfig
+	topic, ok := clowder.KafkaTopics["platform.upload.advisor"]
```

Comment:
Where does this topic name come from?

Reply:
I'm not sure. I thought this was standard for ingress, but that may just be leftover from an older deployment file. What topics should I have access to with ingress?

Reply:
Reworked to match current config.go `platform.payload-status`